index.js

```diff
@@ -0,0 +1,197 @@
+/** @format */
+
+function addItem() {
+  const userInputValue = document.getElementById('input').value;
+
+  const item = {
+    description: userInputValue,
+    completed: false,
+    index: 0,
+  };
```

Comment:
- [ ] Please consider using arrow functions as they make your code clean and one of the objectives of implementing this project is learning how to use ES6 syntax. example: `const addItem = () => {}`

Reply:
Thanks and duly noted